stellargraph/layer/graph_attention.py

```diff
@@ -720,17 +720,19 @@ def __init__(
 
         # Check activations (list of str):
         # check type:
+        if activations is None:
```

Comment:
A better default would be all "elu" (which is the configuration used in the original paper).